dev-packages/browser-integration-tests/suites/integrations/featureFlags/statsig/init.js

```diff
@@ -0,0 +1,30 @@
+import * as Sentry from '@sentry/browser';
+
+class MockStatsigClient {
+  constructor() {
+    this._gateEvaluationListeners = [];
+  }
+
+  on(event, listener) {
+    this._gateEvaluationListeners.push(listener);
+  }
+
+  checkGate(name, defaultVal) {
+    // Note the actual StatsigClient.checkGate does not take a defaultVal.
```

Comment:
Hmmm I would try to keep the mock function signatures the same. What about adding a method like `setMockValue(name, value)` and have `checkGate` read from that?